### ISSUE TITLE
Missileguidance error when trying to lock

### ISSUE THREAD

**Issue by alganthe** [acemod/ACE3#4007]
**Arma 3 Version:** `1.63` dev
**CBA Version:** `2.4.1` stable
**ACE3 Version:** `3.6.0` latest RC

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
- When trying to lock something with the Titan AT the RPT file will be spammed.

**Steps to reproduce:**
- Try to lock something, check the RPT

**Where did the issue occur?**
- SP / MP

**Placed Modules:**
- None

**RPT log file:**
```
if(!(terrainIntersectASL [ _seekerPos, _targe>
 3:44:55   Error position: <terrainIntersectASL [ _seekerPos, _targe>
 3:44:55   Error 0 elements provided, 3 expected
 3:44:55 File z\ace\addons\missileguidance\functions\fnc_checkLos.sqf, line 11
 3:44:55 Error in expression <etPosASL _seeker;
_return = true;

```

**Comment by PabstMirror** [comment]
Issue seems to only be with the new launchers on apex/dev branch.

Should be a simple fix, but also not critical for 3.6 release

**Comment by commy2** [comment]
For now , fixed with #4014. A better fix will likely come later.
